src/org/labkey/test/pages/core/admin/FolderTypePage.java

```diff
@@ -0,0 +1,85 @@
+package org.labkey.test.pages.core.admin;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class FolderTypePage extends LabKeyPage<FolderTypePage.ElementCache>
```

Comment:
Page class should match action name.
```suggestion
public class FolderTypesPage extends LabKeyPage<FolderTypePage.ElementCache>
```

Reply:
Done